gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineClassUniquenessPluginIntegrationTest.groovy

```diff
@@ -0,0 +1,154 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline
+
+import java.nio.file.Files
+import java.util.stream.Stream
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.TaskOutcome
+
+class BaselineClassUniquenessPluginIntegrationTest extends AbstractPluginTest {
+
+    def standardBuildFile = """
+        plugins {
+            id 'java'
+            id 'com.palantir.baseline-class-uniqueness'
+        }
+        subprojects {
+            apply plugin: 'java'
+        }
+        repositories {
+            mavenCentral()
+            maven { url 'https://dl.bintray.com/palantir/releases' }
+        }
+    """.stripIndent()
+
+    def 'Task should run as part of :check'() {
+        when:
+        buildFile << standardBuildFile
+
+        then:
+        def result = with('check', '--stacktrace').build()
+        result.task(':checkClassUniqueness').outcome == TaskOutcome.SUCCESS
+    }
+
+    def 'detect duplicates in two external jars'() {
+        when:
+        buildFile << standardBuildFile
+        buildFile << """
+        dependencies {
```

Comment:
i'd prefer to trim this test case down to a more minimal one

Reply:
Lots of entries here allows me to verify the sorted table works nicely